### PR TITLE
Fix display order on Geolocation Data

### LINKED ISSUE #11741: Get Geolocation Data - Amend running code.
To better display use of Geo-location data. recommend the following amendment. 
[Get Geolocation Data](https://www.freecodecamp.com/challenges/get-geolocation-data)
current: - 
```
</script>
<div id = "data">
  <h4>You are here:</h4>
  
</div>
```
image

Change to: -
```
</script>
    <h4>You are here:</h4>
<div id = "data">
   
</div>
```
image

### PR DESCRIPTION
**Pull request by ubershibs**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [X] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [X] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [X] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [X] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [X] Tested changes locally.
- [X] Closes currently open issue (replace XXXX with an issue no): Closes #11741

#### Description

Switched two lines of code https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/seed/challenges/01-front-end-development-certification/json-apis-and-ajax.json#L500-L501, per the ticket, to make the display of geolocation data more readable.

### CODE REVIEW

**Comment by raisedadead** [comment]
@ubershibs your changes are perfect. thanks a lot for your time and hard work!

Having said that, we would love to let absolute newbies take this one to give them an idea of the processes.
From your profile you have a stunning experience in contributing if you do not mind, can we close this? 

We do NOT intend to discourage you. But I am quite hopeful that you would be interested in taking up the re-wording issue on the other thread?

Do let us know. You would still be happy to accept this PR, though :wink: ?

**Comment by ubershibs** [comment]
Sure yeah, I can tackle the other issue -- Sorry, I guess I considered myself a newbie because I've never actually contributed to anyone's projects but my own before. But thanks for  the confidence boost! Go ahead and close this.

**Comment by raisedadead** [comment]
@ubershibs awesome! thanks and cant wait to see your contributions soon!
